src/app/services/realestate-api.service.ts

```diff
@@ -15,4 +16,8 @@ export class RealestateApiService {
   getAllReferenceData() : Observable<ReferenceDataResponseDto> {
     return this.httpCLient.get<ReferenceDataResponseDto>(this.baseUrl + 'ReferenceData')
   }
+
+  getAllCustomerData() : Observable<Array<CustomerModel>>{
```

Comment:
We should create a new customer-api.service.ts to store it's endpoints
This one is for real estate

Reply:
Done